src/MomentRe.re

```diff
@@ -303,6 +303,12 @@ external momentWithFormats: (string, list(string)) => Moment.t = "moment";
 
 [@bs.module] external momentWithComponents: list(int) => Moment.t = "moment";
 
+[@bs.module "moment"]
+external momentUtcWithFormats: (string, array(string)) => Moment.t = "utc";
```

Comment:
array(string), not list(string) like in momentWithFormats. Because list(string) not convert properli to js array, but reason array convert good. In momentWithFormats is bug i think (not tested) - mabye i'll make another PR to fix it.